setup.py

```diff
@@ -213,7 +213,7 @@ def finalize_options(self, build_ext):
             found_mfc = False
             for incl in os.environ.get("INCLUDE", "").split(os.pathsep):
                 # first is a "standard" MSVC install, second is the Vista SDK.
-                for candidate in (r"..\src\occimpl.h", r"..\..\src\mfc\occimpl.h"):
+                for candidate in ("..\\src\\occimpl.h", "..\\..\\src\\mfc\\occimpl.h"):
```

Comment:
using raw strings before was fine, right? I see no reason to avoid raw strings.

Reply:
Just out of styling/consistency between using r-strings or `\\` for paths strings. If you'd prefer, I can revert those specific manual changes.

Reply:
I don't see a need to enforce consistency here - IMO r-strings exist purely because consistency is over-rated :)

Reply:
I just remembered the other reason, it was visual (though it'll slightly differ per editor)
![image](https://github.com/mhammond/pywin32/assets/1350584/029ed80d-29df-4186-8086-290f43fee71b)
vs
![image](https://github.com/mhammond/pywin32/assets/1350584/81491d49-827b-463f-9bee-463459d7f383)
but yeah that's minor anyway, I'm reverting atm to reduce changes

Reply:
huh, I see that in vscode too. That's gotta be a bug though, right?

Reply:
This would be the intent:
![image](https://github.com/mhammond/pywin32/assets/1350584/588f655e-146d-4916-b59c-b8ad6f8e80ad)
To let you know that those characters will be escaped as part of a regex string.

Reply:
That doesn't make sense to me. The link you pointed to basically says "if you don't want to double up your backslashes when matching them in a regex, use a raw string" - it doesn't seem to come close to saying "raw strings should *only* be used when you want to avoid using 4 backslashes in a regex". Eg, https://docs.python.org/3/reference/lexical_analysis.html says just "such strings are called raw strings and treat backslashes as literal characters." and makes no reference to regular expressions.

Reply:
Not saying it can't be used for anything else than regex strings.
Just to answer your comment about `That's gotta be a bug though, right?`. It's intended semantic highlighting on microsoft's part to help with "the backslash plague" *when* using regex. Similarly to how it's done for JS/TS `/regex/` strings